app.py

```diff
@@ -444,8 +444,8 @@ def get(self):
         else:
             values['account'] = config.DUMMY_VALUES['account']
             values['principal'] = config.DUMMY_VALUES['principal']
-        values['payload_id'] = self.payload_id
-        values['hash'] = self.payload_id  # provided for backward compatibility
+        values['request_id'] = self.request_id
+        values['payload_id'] = self.request_id
```

Comment:
Are we leaving this value in until payload_id is no longer being expected elsewhere?

Reply:
That's the plan. This way, we can get updated but won't bother other apps that haven't made the change. Trying to mess up other people's sprints.